src/layouts/rails-component-layout.tsx

```diff
@@ -313,20 +315,24 @@ export default function RailsComponentLayout({data}) {
             <TableOfContents aria-labelledby="toc-heading" items={tableOfContents.items} />
           </Box>
           <Box>
-            {/* @ts-ignore */}
-            <Note variant="warning">
-              <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>Work in progress</Text>
-              We are currently transferring the Rails documentation for {name} from a different site to this page. To
-              view the original documentation, please visit the{' '}
-              <Link href={railsUrl}>Primer ViewComponents documentation for {name}</Link>.
-            </Note>
-
             <Box sx={{display: 'flex', gap: 2, mb: 4}}>
               <Label size="large">v{data.primerRailsVersion.version}</Label>
               <StatusLabel status={sentenceCase(status)} />
               <AccessibilityLabel a11yReviewed={a11y_reviewed} short={false} />
             </Box>
 
+            {/* @ts-ignore */}
```

Comment:
I don't think we should be using a warning here, as this is knowledge the user should be aware of - not something temporary to warn folks of. 

I'd suggest either going with the blue informational alert or keeping this as plain text at the beginning of the component documentation. It seems like required documentation core to all the forms components.